src/timepicker/timepicker-i18n.ts

```diff
@@ -24,13 +23,11 @@ export abstract class NgbTimepickerI18n {
 
 @Injectable()
 export class NgbTimepickerI18nDefault extends NgbTimepickerI18n {
-	private _periods = getLocaleDayPeriods(inject(LOCALE_ID), FormStyle.Standalone, TranslationWidth.Narrow);
-
 	getMorningPeriod(): string {
-		return this._periods[0];
+		return 'AM';
```

Comment:
Looks like this service has changed from providing localized day period strings to providing non-localized ones.  How should apps relying on the old localized-by-default behavior handle this change?  Are they now going to be expected to implement their own version of this service if they want these to be localized?

Reply:
We can use 
```
$localize`AM`
```
, but I didn't find a language where AM/PM is translated

Reply:
Also I didn't find a way to get it natively from JS

Reply:
> I didn't find a language where AM/PM is translated

In Welsh, it's often "yb"/"yh" (but I think AM/PM can also be used).  I guess some languages that use AM/PM may format it differently (e.g., as "a. m."/"p. m." in Spanish).

> I didn't find a way to get it natively from JS

It is sort of possible in a long-winded way, but not 100% reliable as it relies on the browser having support for the given locale.  E.g.:

```ts
_periods: string[] = (
    const getPeriod = (parts: {type: string, value: string}[], fallback: string) => parts.find((p) => p.type === "dayPeriod")?.value || fallback;

    const df = new Intl.DateTimeFormat(
      inject(LOCALE_ID),
      {
        timeStyle: 'full',
        hourCycle: 'h12',
        timeZone: 'UTC'
      }
    );

    const am = getPeriod(df.formatToParts(new Date(3600000)), 'AM');
    const pm = getPeriod(df.formatToParts(new Date(3600000*13)), 'PM');

    return [am, pm];
)();
```

Not exactly pretty, and relies on the browser having the locale data available.

Alternatively, you could use Angular's `formatDate` function to do something similar, as I believe that function has not been deprecated, and should work for all locales _Angular_ knows about (i.e., it doesn't require browser-provided locale data):

```ts
#locale = inject(LOCALE_ID);
_periods = [
   formatDate(new Date(3600000)), 'a', this.#locale, 'UTC'),
   formatDate(new Date(3600000*13)), 'a', this.#locale, 'UTC'),
];
```